TabLayout/tests/TabGroup-specs.js

```diff
@@ -16,14 +18,14 @@ describe('TabGroup specs', () => {
 			/>
 		);
 
-		const expected = 1;
-		const actual = subject.find('Tab').length;
+		// When one of the tabs doesn't have an icon, the role='group' is not applied to the component
+		const actual = screen.queryByRole('group');
```

Comment:
Instead of this, can we check the below icon? it's more intuitive.
![image](https://user-images.githubusercontent.com/5037429/126580188-5b8dc1ca-748a-4138-b36b-0e37ce866816.png)

Reply:
Done